smartseq2_single_sample/check_reference/buildRef.wdl

```diff
@@ -0,0 +1,20 @@
+import "BuildStarReferenceBundle.wdl" as buildStarRef
+
+workflow buildStarRefs {
+  File ref_metadata_file
+  Array[Object] metadata = read_objects(ref_metadata_file)
+  
+  scatter (record in metadata){
+    String name_ref = record.name1
+    String name2 = record.name2
```

Comment:
Is this not used? Also It would help to have a comment here or a more informative name on this and also name_ref.